core/state_transition.go

```diff
@@ -304,6 +304,12 @@ func (st *StateTransition) TransitionDb() (*ExecutionResult, error) {
 	// 5. there is no overflow when calculating intrinsic gas
 	// 6. caller has enough balance to cover asset transfer for **topmost** call
 
+	// If ETX, bypass accessList enforcement
+	if st.msg.IsETX() {
+		orig := st.state.ConfigureAccessListChecks(false)
```

Comment:
In what case would orig ever be false? Perhaps it makes more sense to intentionally reset it to true every time

Reply:
For context: The `bypassAccessListCheck` variable is just a rename of the `accessListVariable` you had before. I renamed it, because I am using it generally as a way to bypass the checks, instead of just for debugging.

But from the comment you originally left, my understanding is you bypass checks when building the accessList, and in those cases we need to preserve the original state of the variable:
```
accessListDebug bool // used for simulating the EVM to create an access list
```

Reply:
Which is the case here... I am intentionally setting it to false to disable the checks when processing an ETX.

Reply:
Do you have a problem with me setting it back to orig? I don't see how that could break, but _not_ setting it back _would_ break the debug case you just described

Reply:
It's fine for now, but if orig is false in consensus, there is a bug. And even in the create access list case, it's set to debug for every call